portal/templates/flask_user/flask_user_base.html

```diff
@@ -9,7 +9,7 @@
 {% block document_ready %}
 	{{super()}}
 	$(document).on("ready", function() {
-		$("input[type='text']").on("blur", function() {
+		$("input[type='text']").on("blur paste", function() {
 			$(this).val($.trim($(this).val()));
```

Comment:
Ivan, I believe it will just be the username/email and password fields in this case, since flask_user_base.html is only extended by templates within the flask_user folder.  That is why I thought it safe to do that.